### DISCUSSION TITLE
Command to fix indentation?

### DISCUSSION THREAD

**Discussion by markoa** [zed-industries/zed#6585] [Feature Requests]
Is there a way to, after selecting multiple lines of code, fix indentation in the selected area? Similar to [typing `=` in Vim](https://vim.fandom.com/wiki/Fix_indentation).

**Comment by uwiger** [comment]
Yes, I often find myself missing something like the `M-x indent-region`, or even `Ctrl-I` in Emacs.

**Comment by morgoth** [comment]
@markoa Did you find an answer? I'm wondering the same

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
